data/starksheet.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "Starksheet",
+  "description": "Like a spreadsheet, powered by the open web: each cell can call any contract using any on-chain data.",
+  "short_description": "The web3 spreadsheet",
+  "tags": [
+    "Onramp",
+    "DeFi",
+    "NFTs",
+    "Social",
+    "DAOs"
+  ], //Remove all that don't apply
```

```suggestion
   "tags": [
    "NFTs",
    "Social"
  ],
```

We need to remove the comment in order for it to work.  
Also, ideally we have max 2 or 3 tags